Week3/More JavaScript.js

```diff
@@ -42,11 +42,28 @@ let vehicleCategory = {
     1 : "car",
     2 : "motorbike", 
 };
-    function vehicleType(color,category) {
-        console.log("a " + color + " " + vehicleCategory[category] );        
+    function vehicleType(color, category) {
+  if (category <= 2) {
+    console.log('a ' + color + ' ' + vehicleCategory[category]);
+  } else {
+    console.log('Please enter valid');
+  }
 }
-     
-vehicleType("blue", 2);
+vehicleType('blue', 2);
+vehicleType('green', 3);
+
+// I could have set the function vehicleType also with array like this. 
+
+/* let vehicleCategory = ['car', 'motorbike'];
+function vehicleType(color, category) {
+  if (category <= 2) {
```

Comment:
Ah, if you do it this way, you can use `category` <= `vehicleCategory.length` instead of `category` <= `2` :)

Reply:
Right, it is totally a more practical way :)